package/Windows/Includes/DevolutionsGateway.wxi

```diff
@@ -13,8 +13,10 @@
   <?define InfoURL="https://server.devolutions.net" ?>
 
   <!-- SDDL string representing desired %programdata%\devolutions\gateway ACL -->
-  <!-- SYSTEM/BuiltInAdministrators = Full Control, BuiltInUsers - Read/Execute -->
-  <?define ProgramDataSddl="D:PAI(A;OICI;FA;;;SY)(A;OICI;FA;;;BA)(A;OICI;0x1200a9;;;BU)" ?>
+  <!-- Easiest way to generate an SDDL is to configure the required access, and then query -->
+  <!-- the path with PowerShell: `Get-Acl | Format-List`-->
+  <!-- SYSTEM/BuiltInAdministrators = Full Control, LocalService = Read/Write/Execute, BuiltInUsers - Read/Execute -->
+  <?define ProgramDataSddl="D:PAI(A;OICI;FA;;;SY)(A;OICI;0x1201bf;;;LS)(A;OICI;FA;;;BA)(A;OICI;0x1200a9;;;BU)" ?>
```

Comment:
Write once, read never™️
Thank you for the tip in the comment above 😄

Reply:
I thought it would be best, although I wanted you all to believe I was a magician who can conjure SDDL strings from my fingertips 😂